frontend/src/metabase/public/components/EmbedFrame/LogoBadge/LogoBadge.tsx

```diff
@@ -0,0 +1,39 @@
+import cx from "classnames";
+import { t } from "ttag";
+
+import ExternalLink from "metabase/core/components/ExternalLink";
+
+import LogoBadgeStyle from "./LogoBadge.module.css";
+import MetabaseLogoWithText from "./metabase_logo_with_text.svg?component";
+
+export const LogoBadge = ({ dark }: { dark: boolean }) => {
+  const utmContentValue = `embedded_banner_${encodeURIComponent(
+    getHostAppUrlDomain(),
+  )}`;
+
+  return (
+    <ExternalLink
+      className={cx(LogoBadgeStyle.metabaseLink, {
+        [LogoBadgeStyle.dark]: dark,
+      })}
+      href={`https://www.metabase.com/powered-by-metabase?utm_medium=referral&utm_source=product&utm_campaign=powered_by_metabase&utm_content=${utmContentValue}`}
+      target="_blank"
+    >
+      <span>{t`Powered by`}</span>
+      {/* eslint-disable-next-line no-literal-metabase-strings -- This embedding badge which we don't want to show the whitelabeled name */}
+      <MetabaseLogoWithText height={32} aria-label="Metabase" />
```

Comment:
`aria-label` ensures we can still find `link` with name: `Powered by Metabase` in E2E tests.